dxp/architecture/wysiwyg.md

```diff
@@ -0,0 +1,157 @@
+# Rich Text Editors
```

Comment:
This file seems a copy of the other that looks like you meant to delete, so I didn't read it.

Reply:
Yeah... meant to move it... somehow ended up with 2 🙈